schema/private.openapi.yaml

```diff
@@ -315,6 +336,36 @@ components:
       - org_id
       - hosts
 
+    JobRequestBody:
+      type: object
+      properties:
+        org_id:
+          $ref: '#/components/schemas/OrgId'
+        principal:
+          $ref: '#/components/schemas/Principal'
+        url:
+          $ref: './public.openapi.yaml#/components/schemas/Url'
+        playbook_name:
+          $ref: './public.openapi.yaml#/components/schemas/PlaybookName'
+        hosts:
+          type: array
+          items:
+            description: Identifies a record of the Host-Inventory service
+            type: string
+            minLength: 1
+        web_console_url:
+          $ref: './public.openapi.yaml#/components/schemas/WebConsoleUrl'
+        labels:
+          $ref: './public.openapi.yaml#/components/schemas/Labels'
+        timeout:
+          $ref: './public.openapi.yaml#/components/schemas/RunTimeout'
+      required:
+      - org_id
+      - principal
+      - hosts
+      - url
+      - playbook_name
```

Comment:
The design doc only mentions the requirement of `hosts` and the prefix for this endpoint. I diverged a little bit and added a few more params here. Some of these, such as `org_id`, are pretty basic, which some are not e.g. `labels`, `web_console_url`.
Wanted to make sure that sounds good to everyone. I don't think the app teams will reach out to other services to provide us these additional parameters. I think they should be able to set these themselves while requesting a playbook run.